dashboard/app/models/lesson_group.rb

```diff
@@ -227,4 +227,25 @@ def i18n_hash
       {}
     end
   end
+
+  def copy_to_script(destination_script, new_level_suffix = nil)
+    return if script == destination_script
+    raise 'Both lesson group and script must be migrated' unless script.is_migrated? && destination_script.is_migrated?
+    raise 'Destination script and lesson group must be in a course version' if destination_script.get_course_version.nil? || script.get_course_version.nil?
+
+    ActiveRecord::Base.transaction(requires_new: true, joinable: false) do
```

Comment:
could we forego this transaction? it looks like it is always done inside clone_migrated_script, which has its own transaction.